src/ServiceControlInstaller.Engine/Instances/V6Transports.cs

```diff
@@ -50,6 +50,14 @@ public static TransportInfo FindByName(string name)
                 TypeName = "NServiceBus.RabbitMQTransport, NServiceBus.Transports.RabbitMQ",
                 MatchOn = "NServiceBus.RabbitMQ",
                 SampleConnectionString = "host=<HOSTNAME>;username=<USERNAME>;password=<PASSWORD>"
+            },
+            new TransportInfo
+            {
+                Name = "AmazonSQS",
+                TypeName = "NServiceBus.SqsTransport, NServiceBus.AmazonSQS",
+                MatchOn = "NServiceBus.AmazonSQS",
+                SampleConnectionString = "AccessKeyId=<ACCESSKEYID>;SecretAccessKey=<SECRETACCESSKEY>;Region=<REGION>",
+                Help = "AccessKeyId will be promoted to AWS_ACCESS_KEY_ID, SecretAccessKey to AWS_SECRET_ACCESS_KEY and Region to AWS_REGION environment variable."
```

Comment:
Only for the current execution of the process. Rather than saying they will be promoted, can we say:

> If AccessKeyId is omitted then ServiceControl will use the value of the AWS_ACCESS_KEY_ID environment variable.
> If SecretAccessKey is omitted then ServiceControl will use the value of the AWS_SECRET_ACCESS_KEY environment variable.

Reply:
Oh. It doesn't fall back any more. So even if I have these values in environment variables SC won't use them?

Reply:
Correct. We couldn't get it working with they way the process runs. We
could only supporr it if we enforce system wide env variables which seems a
bit hard. That's why we take the mandatory input and promote it to process
variables